Gopkg.lock

```diff
@@ -19,7 +19,7 @@
     "aws/endpoints",
     "internal/ini",
     "internal/sdkio",
-    "internal/shareddefaults",
+    "internal/shareddefaults"
```

Comment:
This seems like a spurious and unnecessary change.  Do you know where it came from?

Reply:
This was done automatically up running `dep ensure`